Makefile

```diff
@@ -8,16 +8,22 @@ build:
 clean:
 	docker-compose down -v --remove-orphans
 
+debug:
+	docker-compose stop app ; docker-compose run --service-ports app
```

Comment:
```suggestion
        # Stop the canonical app container to avoid a port collision. Use `run`
        # so that we have interactive console access for the debugger.
	docker-compose stop app ; docker-compose run --service-ports app
```